src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -1019,4 +1019,7 @@ Do you agree to the terms?</value>
   <data name="PromptOptionYes" xml:space="preserve">
     <value>Yes</value>
   </data>
-</root>
+  <data name="InstallFlowRegistrationDeferred" xml:space="preserve">
+    <value>Successfully installed. Restart the application to complete the upgrade.</value>

```

Comment:
this only applies to upgrade?

Reply:
I think if the package is in use, then it could only be "upgrading"

Reply:
Any install action on MSIX uses this path; I used the verb upgrade to stay consistent with our own command but it could happen on install or import as well.